godot_project/project_workspace/workspace_context/workspace_context.gd

```diff
@@ -626,15 +624,14 @@ func end_simulation_if_running() -> void:
 
 ## This method is automatically called when any code attempts
 ## to create a new action with `snapshot_moment()`
-func apply_simulation_if_running() -> void:
+func _apply_simulation_if_running() -> void:
```

Comment:
Calling this directly will bypass the `about_to_apply_simulation` signal and break the emitters. So this is private now.